docs/application/web/guides/connectivity/download.md

```diff
@@ -85,25 +85,45 @@ To provide the user access to Internet resources, you must learn how to manage d
 
    The `start()` method returns a unique identifier for the download operation.
 
-5. During the download:
+   You can check the status of the download operation by calling the `getState()` method with the download ID:
+
+      ```
+      tizen.download.getState(downloadId);
+      ```
+
+5. During the download operation, you can modify the download state.
+
+    >**Note**
+    >
+    >Depending on the download size and the connection speed, the download operation completes almost instantly.
 
    1. To pause the download, use the `pause()` method with the download ID:
 
       ```
       tizen.download.pause(downloadId);
       ```
+        If the current state is DOWNLOADING or QUEUED, calling the `pause()` method changes the download state to PAUSED.
 
    2. To resume the download, use the `resume()` method with the download ID:
 
       ```
       tizen.download.resume(downloadId);
       ```
+        If the current state is PAUSED, CANCELED, or FAILED, calling the `resume()` method changes the download state to DOWNLOADING or QUEUED.
 
    3. To cancel the download, use the `cancel()` method with the download ID:
 
       ```
       tizen.download.cancel(downloadId);
       ```
+        If the current state is DOWNLOADING or QUEUED, calling the `cancel()` method changes the download state to CANCELED.
+
+   4. To abandon the download, use the `abandon()` method with the download ID:
+
+      ```
+      tizen.download.abandon(downloadId);
+      ```
+        The abandoned download operation frees its resources and stops further download operations.
```

Comment:
I am not sure if the download operation frees its resources, there is a problem with subject of this sentence. Moreover, this comment suggest that 'abandon' stops download operationS - not only this one. I suggest rephrasing this comment to:
For the abandoned download operation, all its resources are released. After that, operation cannot be resumed.

Reply:
@pkosko I too was sceptic about the same, refer to https://github.com/Samsung/tizen-docs/pull/827#discussion_r327441130 . 

Just a suggestion: 
How about using this? 

For the abandoned download operation since all its resources are already released thus, download operation cannot be resumed. 
@mijin85cho 
Please share your opinion on this.

Reply:
"For any abandoned download operation, the resources are already released. Thus the download operation cannot be resumed." I ok for me.

Reply:
Done